src/ProjectService.Mappers/ModelsMappers/Interfaces/IProjectInfoMapper.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Models;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.ModelsMappers.Interfaces
+{
+    [AutoInject]
+    public interface IProjectInfoMapper
+    {
+        ProjectInfo Map(DbProject dbProject, IGetDepartmentResponse departmet);
```

Comment:
I would only send the name of the department and not the entire structure

Reply:
Fixed